docs/stories/components/Checkbox.stories.js

```diff
@@ -25,21 +51,112 @@ function CheckboxStory() {
           value={val}
         />
       </div>
+      <div style={{ marginTop: '-5px', marginBottom: 20 }}>
+        <ul>
+          <Checkbox
```

Comment:
It's better if the Checkbox is not a `ul` child so outside or in a `li` (to change is the `Pre` too)